exporter/kafkaexporter/config.modified.go

```diff
@@ -53,6 +53,9 @@ type Config struct {
 
 	// Compression defines the compression method and compression level, if applicable.
 	Compression Compression `mapstructure:"compression"`
+
+	// Debug mode. Log info for spans that exceed Producer.MaxMessageBytes early.
+	DebugMode bool `mapstructure:"debug_mode"`
```

Comment:
nit: `Mode` is redundant.  maybe just `debug` true:false

Reply:
sure.

Reply:
done